cpp/ps0003/p3Solution.cpp

```diff
@@ -9,11 +9,13 @@ void checkEvenorOdd(int num) {
 }
 
 int main() {
+	// prompting user for number 2/2
 	int num;
 	cout << "Enter a number to check if it's even or odd: ";
 	cin >> num;
-	checkEvenorOdd(num);
 
+	// checking if the number is odd or even 3/3
+	checkEvenorOdd(num);

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console